src/api/hooks/useApiQuery.ts

```diff
@@ -12,16 +12,22 @@ function useApiQuery<TData>(
   options: UseQueryOptions<TData> = {enabled: true},
 ) {
   const {api} = useApi();
+  const network = api?.runtimeChain != null ? api.runtimeChain.toString() : undefined;
```

Comment:
You can also do: 
  `const {currentNetwork} = useContext(NetworkContext);
`

Reply:
`const {currentNetwork} = useContext(NetworkContext)`
I gave it a shot, but for some reason it does not behave as expected(refetch queries) for the screens in the drawer stack.

@franjs 
For now I am keeping it as it is. If you have time please ping me to discuss.

Reply:
I'm not sure I follow, when we change the network we are actually changing the state in `NetworkContext` so it must be different, the issue is something else